build.sh

```diff
@@ -15,13 +15,20 @@ $GOPATH/bin/govendor sync
 if [[ ! -x $GOPATH/bin/go-bindata ]]; then
   go get -u github.com/jteeuwen/go-bindata/...
 fi
-$GOPATH/bin/go-bindata -pkg registry -o registry/schema.bindata.go schema
 
-# Until 76f26d79b1be670 gets merged to master.
-if (git rev-list master | grep 76f26d79b1be670) > /dev/null; then
+modtime=$(git log -n 1 --date=raw --pretty=format:%cd -- schema/ | cut -d' ' -f1)
+$GOPATH/bin/go-bindata -modtime "${modtime}" -pkg registry -o registry/schema.bindata.go schema
+
+# Determine the default branch reference for registry/github.go
+SCHEMA_LAST_COMMIT=${SCHEMA_LAST_COMMIT:-$(git log -n 1 --pretty=format:%H -- schema/ registry/schema.bindata.go)}
+if (git rev-list origin/master | grep "${SCHEMA_LAST_COMMIT}") > /dev/null; then
+  # Found no changes for resource template/schema on HEAD.
+  # so that set preference to the master branch.
   LDFLAGS="${LDFLAGS} -X 'registry.GithubDefaultRef=master'"
 else
-  LDFLAGS="${LDFLAGS} -X 'registry.GithubDefaultRef=generalize-template'"
+  # Found resource template/schema changes on this HEAD. Switch the default reference branch.
+  # Check if $GIT_BRANCH has something once in case of running in Jenkins.
+  LDFLAGS="${LDFLAGS} -X 'registry.GithubDefaultRef=${GIT_BRANCH:-$(git rev-parse --abbrev-ref HEAD)}'"
 fi
```

Comment:
Same thing here. Isn't it way more simple to use the local source code when needed?